app/controllers/solidus_stripe/intents_controller.rb

```diff
@@ -8,12 +8,14 @@ def confirm
       begin
         if params[:stripe_payment_method_id].present?
           intent = stripe.create_intent(
-            (current_order.total * 100).to_i,
+            100,
```

Comment:
what's wrong with preauthorizing the full amount?

Reply:
I think that would block twice the full amount on the customer card. 

Also, as a customer, I would not be particularly scared by noticing a $1 pre-authorization charge notice (I've seen something similar done in the past by PayPal and Amazon as well if I recall correctly), but seeing the double amount I would instantly think there's been some double charge error.

Reply:
yep, but if you have $1 on your card, it's not guaranteed that you have the full amount to pay the order. Anyway, I'm ok with this strategy, and we can revisit it later. We just need to be clear in the README that this is the behavior of the gem. Also, what about pre-authorizing $0.01 instead?

Reply:
Unfortunately, you can't pre-authorize less than 0.50$ (or 0.50€... so why not 0.50? if you're in EU and you're buying in USD, the conversion would be 0.43€ which is not acceptable again).
I'm not sure what happens when you don't have enough funds for the order, but I guess that at that point the second payment would fail (for sure the capture would fail). 

Anyway, the second payment is Solidus' default process, for example what happens with Solidus Stripe and Elements. The failed pre-authorization for the whole amount would behave just like in that case.